board/drivers/llcan.h

```diff
@@ -80,6 +79,5 @@ void llcan_init(CAN_TypeDef *CAN_obj) {
 void llcan_clear_send(CAN_TypeDef *CAN_obj) {
   CAN_obj->TSR |= CAN_TSR_ABRQ0;
   CAN_obj->MSR &= ~(CAN_MSR_ERRI);
-  CAN_obj->MSR = CAN_obj->MSR;
```

Comment:
Don't remove this! It clears the register

Reply:
cppcheck complains about: 
`Redundant assignment of 'CAN_obj->MSR' to itself.`